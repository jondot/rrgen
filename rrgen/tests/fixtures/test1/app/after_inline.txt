
// This should be injected with world after hello

pub struct Hello {
  pub name: String,
}