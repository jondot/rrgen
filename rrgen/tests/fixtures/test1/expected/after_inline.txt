
// This should be injected with world after hello

pub struct HelloWorld {
  pub name: String,
}